src/Schema/Type/Enum.php

```diff
@@ -116,10 +92,10 @@ public function setValues(array $values): self
 
     /**
      * @param $key
-     * @param null $val
+     * @param mixed|null $val
```

Comment:
```suggestion
     * @param mixed $val
```
Mixed is inherently nullable